script/Arcueid/Day 2/02_00_ARC02_5.txt

```diff
@@ -37,7 +38,7 @@ Besides, I'm not allowed to go out after ten o'clock.
 [sha:60c37860bedad2e054a77a0717156fca9e42174c]{
 -- Page 330, Offset 3475.
 -- こっそりスタンドライトでも買ってくるとか？」
-Maybe I can sneak out and get a stand light?"
+Maybe I could buy and smuggle in a stand light?"
```

Comment:
revert. I also think we should just use "lamp" instead of the waseieigo term

Reply:
Agree on lamp but I think the prev one is subtly incorrect? Or can at least be read that way. I read this line in JP as "buy in secret", whereas the old TL makes it sound like he's going to sneak out _right now_ and get one from somewhere else in the mansion.

Reply:
hmm, I don't really get that from "sneak out and get a stand light". My preference for the original is just because I feel like it reads more naturally. "could buy and smuggle in" is almost coarse. I would be fine with _just_ "smuggle in".

Reply:
Works for me